### ISSUE TITLE
NullPointerException - ClearConsumer

### ISSUE THREAD

**Issue by mibby** [Rsl1122/Plan-PlayerAnalytics#159]
Plan v3.5 release build.

```
[Jul 09, 04:56:44] main.java.com.djrapitops.plan.data.cache.queue.ClearConsumer Caught java.lang.NullPointerException
[Jul 09, 04:56:44]   com.djrapitops.javaplugin.utilities.Verify.nullCheck(Verify.java:215)
[Jul 09, 04:56:44]   com.djrapitops.javaplugin.utilities.player.Fetch.getPlayer(Fetch.java:65)
[Jul 09, 04:56:44]   main.java.com.djrapitops.plan.data.cache.DataCacheHandler.clearFromCache(DataCacheHandler.java:423)
[Jul 09, 04:56:44]   main.java.com.djrapitops.plan.data.cache.queue.ClearConsumer.consume(DataCacheClearQueue.java:79)
[Jul 09, 04:56:44]   main.java.com.djrapitops.plan.data.cache.queue.ClearConsumer.consume(DataCacheClearQueue.java:61)
[Jul 09, 04:56:44]   main.java.com.djrapitops.plan.data.cache.queue.Consumer.run(Consumer.java:33)
[Jul 09, 04:56:44]   com.djrapitops.javaplugin.task.runnable.RslRunnableFactory$1.run(RslRunnableFactory.java:34)
[Jul 09, 04:56:44]   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:58)
[Jul 09, 04:56:44]   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
[Jul 09, 04:56:44]   com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[Jul 09, 04:56:44]   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[Jul 09, 04:56:44]   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[Jul 09, 04:56:44]   java.lang.Thread.run(Thread.java:748)
[Jul 09, 04:56:44] 
```

**Comment by mibby** [comment]
Seems to happen often.
http://paste.ubuntu.com/25053250/

**Comment by Rsl1122** [comment]
Fixed in 3.5.1

**Comment by mibby** [comment]
Any ETA on release?
